src/number_theory/number_field/embeddings.lean

```diff
@@ -123,3 +125,141 @@ end
 end bounded
 
 end number_field.embeddings
+
+section place
+
+variables {K : Type*} [field K] {A : Type*} [normed_division_ring A] [nontrivial A] (φ : K →+* A)
+
+/-- An embedding into a normed division ring defines a place of `K` -/
+def number_field.place : absolute_value K ℝ :=
+(is_absolute_value.to_absolute_value (norm : A → ℝ)).comp φ.injective
+
+@[simp]
+lemma number_field.place_apply (x : K) : (number_field.place φ) x = norm (φ x) := rfl
+
+end place
+
+namespace number_field.complex_embedding
+
+open complex number_field
+
+open_locale complex_conjugate
+
+variables {K : Type*} [field K]
+
+/-- The conjugate of a complex embedding as a complex embedding. -/
+def conjugate (φ : K →+* ℂ) : K →+* ℂ := star φ
+
+@[simp]
+lemma conjugate_coe_eq (φ : K →+* ℂ) (x : K) : (conjugate φ) x = conj (φ x) := rfl
+
+lemma place_conjugate_eq_place (φ : K →+* ℂ) (x : K) : place (conjugate φ) x = place φ x :=
+by simp only [place_apply, norm_eq_abs, abs_conj, conjugate_coe_eq]
```

Comment:
This is more general without the `x`:
```suggestion
lemma place_star (φ : K →+* ℂ) : place (star φ) = place φ :=
by { ext, simp only [place_apply, norm_eq_abs, abs_conj, conjugate_coe_eq] }
```
(untested)

Reply:
Indeed. It seems to me that in `mathlib`  some of this kind of results are stated as equalities of functions and some as equalities of functions evaluated at an arbitrary point. Is there a general rule? I guess the more general case is the equality of functions but sometimes the other point of view is more convenient...

Reply:
> lemma place_star

Well, you seem to prefer the `star` point of view but, if possible, I would like to keep the terminology of `conjugate` even if it is indeed `star`

Reply:
Generally `f = g` is always a better statement of `f x = g x`, if it's true in that form. The applied form is only usually worth having if the choice is between `f = g.comp h` and `f x = g (h x)`, as here the RHS isn't syntactically the same any more.